tests/tensorflow/test_tensorflow2_model_export.py

```diff
@@ -742,6 +746,34 @@ def test_categorical_model_can_be_loaded_and_evaluated_as_pyfunc(
         results = pyfunc_wrapper.predict(inp_list)
 
 
+@pytest.mark.large
+def test_pyfunc_serve_and_score(saved_tf_iris_model):
```

Comment:
I forgot to add a serving test for tensorflow in #4637.

Reply:
How does this differ from `test_model_deployment_with_default_conda_env`?

Reply:
I think we only need one of these, preferably `pyfunc_serve_and_score_model`. Can we remove the sagemaker scoring test case and also remove similar sagemaker scoring test cases from other flavors?

Reply:
Ok, I’ll remove sagemaker scoring tests. If I’m not mistaken, we no longer run them

Reply:
Sounds good!

Reply:
Can I address this as a follow-up?

Reply:
Yep, I didn't realize that the `release` tag was on all of those. Feel free to file an internal ticket for us to address this later. No need to follow up now :)

Reply:
@dbczumar Filed a PR: https://github.com/mlflow/mlflow/pull/4649

Reply:
> Yep, I didn't realize that the release tag was on all of those. Feel free to file an internal ticket for us to address this later. No need to follow up now :)

Got it!